schemas/2018-08-20-preview/Microsoft.HealthcareApis.json

```diff
@@ -108,61 +108,27 @@
       ],
       "description": "An access policy entry."
     },
-    "ServiceCosmosDbConfigurationInfo": {
-      "type": "object",
-      "properties": {
-        "offerThroughput": {
-          "oneOf": [
-            {
-              "type": "integer"
-            },
-            {
-              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
-            }
-          ],
-          "description": "The provisioned throughput for the backing database."
-        }
-      },
-      "description": "The settings for the Cosmos DB database backing the service."
-    },
     "ServiceAuthenticationConfigurationInfo": {
       "type": "object",
       "properties": {
         "authority": {
-          "oneOf": [
-            {
-              "type": "string",
-              "description": "The authority url for the service"
-            },
-            {
-              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
-            }
-          ],
-          "readOnly": false
+          "type": "string",
+          "description": "The authority url for the service"
         },
         "audience": {
-          "oneOf": [
-            {
-              "type": "string",
-              "description": "The audience url for the service"
-            },
-            {
-              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
-            }
-          ],
-          "readOnly": false
```

Comment:
Why is "readOnly" property removed here? Is the value of readonly false by default?

Reply:
`readOnly` isn't a valid annotation as per the JSON schema 4 spec. Read-only swagger properties are simply removed for template generation.